tests/Unit/Product/GlobalProductTest.php

```diff
@@ -206,4 +274,99 @@ public function testItReturnsAJsonRepresentation(): void
 
         $this->assertJsonStringEqualsJsonString(Json::encode($expectedJson), Json::encode($product));
     }
+
+    /**
+     * @dataProvider invalidXmlStructure
+     */
+    public function testItThrowsAExceptionWithoutASellerSkuInTheXml($property): void
+    {
+        $xmlString = sprintf(
+            $this->getSchema('Product/GlobalProduct.xml'),
```

Comment:
The xml string creation here is the same than the assignment on testItMakesAProductFromXml, can you create a method to assign and return that?

Reply:
I've refactored this file and the product test with the same change. Thanks!